openhands/llm/llm.py

```diff
@@ -400,8 +405,7 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
                 assert self.config.log_completions_folder is not None
                 log_file = os.path.join(
                     self.config.log_completions_folder,
-                    # use the metric model name (for draft editor)
-                    f'{self.metrics.model_name.replace("/", "__")}-{time.time()}.json',
+                    f'{self.model_name.replace("/", "__")}-{time.time()}.json',
```

Comment:
```suggestion
                    f'{self.config.model.replace("/", "__")}-{time.time()}.json',
```
Probably? We have to have model at this point, we likely don’t actually need to send a model named “default”, WDYT?